.github/workflows/release.yml

```diff
@@ -150,3 +97,27 @@ jobs:
           vercel-args: '--prod'
           scope: ${{ secrets.VERCEL_SCOPE }}
           working-directory: packages/test-app/dist
+
+  publish_npm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Prepare repository
+        run: git checkout master
+      - name: Set Node Version
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12.16.1
+      - name: Restore lerna cache
+        id: lerna-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Install monorepo deps
+        run: yarn --frozen-lockfile
+      - name: Publish
```

Comment:
Looks like this workflow is going to run at the same time as the `Semantic Release` workflow, meaning this publish step may sometimes beat the other workflow's `version` step, and vice versa through a race condition. We should probably put the `version` and `publish` steps in sequential order in the same workflow to eliminate this race condition, thoughts?

Reply:
See my other comment - I don't think I want to combine them. Both workflows will never run on the same commit - one workflow only runs with a new tag, and other only runs if there _isn't_ a tag.